### PR TITLE
Feature follow company

### PR DESCRIPTION
**Pull request by LinhHoang1110**
## Related Tickets:
- [#28764](https://edu-redmine.sun-asterisk.vn/issues/28764)

## WHAT this PR does?
- Follow các công ty có trong DB

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by LinhHoang1110** [comment]
ready

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -68,6 +68,7 @@ protected function create(array $data)
             'name' => $data['name'],
             'email' => $data['email'],
             'password' => Hash::make($data['password']),
+            'role_id' => 4,
```
Comment (thaonp104) [inline app/Http/Controllers/Auth/RegisterController.php]: cho number vào config

**Review by huyenttm-0901 — COMMENTED** [review]
fix comments

**Comment by LinhHoang1110** [comment]
ready

**Pull request merged**